specs/ClearBrowsingData.md

```diff
@@ -247,22 +250,22 @@ namespace Microsoft.Web.WebView2.Core
 {
     [Flags] enum CoreWebView2BrowsingDataKinds
     {
-        AppCache = 1,
-        FileSystems = 2,
-        IndexedDb = 4,
-        LocalStorage = 8,
-        WebSql = 16,
-        CacheStorage = 32,
-        AllDomStorage = 64,
-        Cookies = 128,
-        AllSite = 256,
-        DiskCache = 512,
-        DownloadHistory = 1024,
-        GeneralAutofill = 2048,
-        PasswordAutosave = 4096,
-        BrowsingHistory = 8192,
-        Settings = 16384,
-        AllProfile = 32768,
+        FileSystems = 1,
+        IndexedDb = 2,
+        LocalStorage = 4,
+        WebSql = 8,
+        CacheStorage = 16,
+        AllDomStorage = 32,
+        Cookies = 64,
+        AllSite = 128,
+        DiskCache = 256,
+        DownloadHistory = 512,
+        GeneralAutofill = 1024,
+        PasswordAutosave = 2048,
+        BrowsingHistory = 4096,
+        Settings = 8192,
+        AllProfile = 16384,
+        ServiceWorkers = 32768,
```

Comment:
Since we have reorganized enum of CoreWebView2BrowsingDataKinds, should we move ServiceWorkers under WebSql?

Reply:
> Fyi, it's OK to specify with hex syntax, easier to read :)

I agree. It could be more clear, but I just want to be consistent with the previous version.

Reply:
> Since we have reorganized enum of CoreWebView2BrowsingDataKinds, should we move ServiceWorkers under WebSql?

Perhaps we couldn't. 

Since we didn't really reorganize the numbers, instead of keeping consistent with the  actual implementation. Reference: https://learn.microsoft.com/en-us/dotnet/api/microsoft.web.webview2.core.corewebview2browsingdatakinds?view=webview2-dotnet-1.0.1549-prerelease.

Therefore, we can only add `ServiceWorkers` at the end.